libs/sdk-backend-spi/src/workspace/dateFilterConfigs/index.ts

```diff
@@ -1,13 +1,14 @@
 // (C) 2019-2022 GoodData Corporation
+import { IDateFilterConfig } from "@gooddata/sdk-model";
+
 import { IPagedResource } from "../../common/paging";
-import { IDateFilterConfig } from "./types";
 
 /**
  * This service provides access to workspace date filter configs (also known as extended date filters).
  * Date filter configs allow to define your own date filter presets, that appear in the date filter.
  * To make date filter configs work, you have to set enableKPIDashboardExtendedDateFilters feature flag to true.
  *
- * @public
+ * @alpha
```

Comment:
Is this change from public to alpha intentional? I think we should not do that.

Reply:
Just revert of this: https://github.com/gooddata/gooddata-ui-sdk/commit/20469cf8480ad6439eb3942be7e8e8e72b3e1004#diff-80043afc2d6135cf782dcf6b774e080eb87925d7d88b93b1a57ecc4d166271ffL10